Q/QhullMiniWrapper/build_tarballs.jl

```diff
@@ -0,0 +1,32 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "QhullMiniWrapper"
+version = v"1.0.0"
+
+# Collection of sources required to build
+sources = [
+    DirectorySource("./Wrapper")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/
+${CC} -fPIC -shared -o ${libdir}/qhullwrapper.${dlext} qhullwrapper.c -L${libdir} -lqhull_r
+"""
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("qhullwrapper", :qhullwrapper)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = Dependency[
+    Dependency(PackageSpec(name="Qhull_jll"), compat="8")
```

Comment:
You have to use a version which actually exists: https://docs.binarybuilder.org/stable/reference/#BinaryBuilderBase.Dependency

> The optional keyword argument `compat` can be used to specify a string for use in the `Project.toml `of the generated Julia package. If `compat` is non-empty and `build_version` is not passed, the latter defaults to the minimum version compatible with the `compat` specifier.

```suggestion
dependencies = [
    Dependency(PackageSpec(name="Qhull_jll"), compat="8.0.1000")
```